docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -402,17 +402,104 @@ feasible based on the data present, it is also a reasonable approach.
+particularly in the field of health metrics sciences, our goal is often to uncover and scrutinize the 
+underlying factors contributing to health disparities.
+
+One helpful framework for differentiating ways of communicating about race is that of *race-based research vs. race-conscious research* 
+[Cerdena_2020]_. Race-based research, which is characterized 
+by race essentialism, the notion that race is a biological factor (rather than a sociopolitical construct). 
+The practice of using race in this way has historically been the norm in US health research, and causes harm for 
+historically marginalized and minoritized groups, thereby exacerbating the very health inequities their health 
+research may be attempting to elucidate and alleviate. For examples of outdated race-based practices in medicine, 
+and their race-conscious alternatives, see Table 1 from Cerdena et al.’s 2020 paper [Cerdena_2020]_. 
+Race-conscious research, on the other hand, calls for a shift in focus from race to racism as a key determinant of 
+illness and health.
+
+Key recommendations from other sources
+--------------------------------------
+
+Here we have collected recommendations from various experts and guides advocating for new standards of communication 
+regarding racial health disparities.
+
+1. **Use a health equity lens when framing information about health disparities** [Calanan_2023]_.
+
+    Rather than allowing dominant narratives around race, individualism, and meritocracy limit our understanding of 
+    the root causes of health inequities, it is important to center a health equity perspective in health metrics 
+    research.  An equity lens allows us to recognize that systems of power and oppression (including white supremacy) 
+    shape institutional policies and living conditions that systematically harm populations 
+    [AMA_2021]_.
+
+2. **Use person-first language, avoid unintentional blaming, and be as specific as possible about the group to which you're referring**. 
+    
+    For more elaboration on these tips, please see Advancing Health Equity: A Guide to Language, Narrative and Concepts 
+    [AMA_2021]_.
+
+3. **Explicitly define race during experimental design, and specify the reason for its use in the study** [Boyd_2020]_. 
+    
+    As is described in our earlier section, `1.1 Research question`_, you should review all relevant 
+    social, environmental, and structural factors for which race may serve as a proxy measure when defining 
+    race as it pertains to your research,. "For the reader, these additional details enable careful interpretation 
+    of study results and implications. But for authors, it engenders critical thinking about racial constructs that 
+    prevent the reification of race as a biological entity." [Boyd_2020]_ 
+
+    Health metrics researchers should embrace a critical race theory (CRT) framework, which provides "tools for conducting research and practice 
```

Comment:
The word "should" here makes me wonder who is making this recommendation and why CRT is **the** thing that we should embrace. Is this your opinion that we should embrace CRT, or did you find that recommendation in one of your sources? Would it work to say "can embrace" instead, or do you want to be more forceful than that?